apps/api/README.md

```
@@ -0,0 +1,111 @@
+### Test coverage
+
+- **32 tests** covering both endpoints and integration
+- Contact endpoint: 10 tests
+- Newsletter endpoint: 15 tests
+- Integration: 7 tests
+
+See [TESTING.md](./TESTING.md) for detailed testing documentation and [TEST-SUMMARY.md](./TEST-SUMMARY.md) for the latest test results.
+
+## API Endpoints
+
+### POST /api/contact
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)